src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/SchedulesIntTest.kt

```diff
@@ -36,6 +36,7 @@ class SchedulesIntTest : SqsIntegrationTestBase() {
       await untilAsserted {
         NomisApiExtension.nomisApi.verify(
           putRequestedFor(urlEqualTo("/activities/$NOMIS_CRS_ACTY_ID/schedule"))
+            .withRequestBody(matchingJsonPath("id", equalTo("$NOMIS_CRS_SCH_ID")))
```

Comment:
When updating a single course schedule send its id as found from the mapping API